legend-sdlc-test-utils/pom.xml

```diff
@@ -0,0 +1,181 @@
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.sdlc</groupId>
+            <artifactId>legend-sdlc-language-pure-compiler</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.sdlc</groupId>
+            <artifactId>legend-sdlc-protocol-pure</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-test-runner-shared</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-test-runner-mapping</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-language-pure-dsl-service</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.pure</groupId>
+            <artifactId>legend-pure-runtime-java-engine-compiled</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-test-runner-service</artifactId>
+        </dependency>
+        <!-- SDLC -->
+        <!-- PURE-->
+        <dependency>
+            <groupId>org.finos.legend.pure</groupId>
+            <artifactId>legend-pure-m3-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.pure</groupId>
+            <artifactId>legend-pure-runtime-java-extension-external-json</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.pure</groupId>
+            <artifactId>legend-pure-runtime-java-extension-dsl-mapping</artifactId>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.pure</groupId>
+            <artifactId>legend-pure-code-java-compiled-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.pure</groupId>
+            <artifactId>legend-pure-m2-dsl-mapping</artifactId>
+        </dependency>
+        <!-- PURE-->
+        <!-- ENGINE -->
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-language-pure-compiler</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-executionPlan-generation</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-shared-core</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>com.zaxxer</groupId>
+                    <artifactId>HikariCP</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
```

Comment:
Can we get rid of this dependency?

Reply:
`[INFO] 
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce) @ legend-sdlc-test-utils ---
[WARNING] 
Dependency convergence error for com.zaxxer:HikariCP:2.7.6 paths to dependency are:
+-org.finos.legend.sdlc:legend-sdlc-test-utils:0.33.1-SNAPSHOT
  +-org.finos.legend.sdlc:legend-sdlc-language-pure-compiler:0.33.1-SNAPSHOT
    +-org.finos.legend.engine:legend-engine-shared-core:2.29.1
      +-org.finos.legend.shared:legend-shared-pac4j-kerberos:0.11.1
        +-org.finos.legend.shared:legend-shared-pac4j:0.11.1
          +-org.pac4j:dropwizard-pac4j:3.0.0
            +-org.pac4j:pac4j-config:3.0.0
              +-com.zaxxer:HikariCP:2.7.6
and
+-org.finos.legend.sdlc:legend-sdlc-test-utils:0.33.1-SNAPSHOT
  +-org.finos.legend.engine:legend-engine-test-runner-service:2.29.1
    +-org.finos.legend.engine:legend-engine-executionPlan-execution-store-relational:2.29.1
      +-org.finos.legend.engine:legend-engine-executionPlan-execution-store-relational-connection:2.29.1
        +-com.zaxxer:HikariCP:3.4.5
[WARNING] Rule 1: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Rule 1: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:


Failed while enforcing releasability. See above detailed error message.
`